src/php/class-admin-options.php

```diff
@@ -171,12 +171,12 @@ function lazyload_settings_page()	{ ?>
 				<br><span class="claim" style="font-size:15px;font-style:italic;position:relative;top:-7px;"><?php esc_html_e( 'Speed up your site and customise your video player!', LL_TD ); ?></span>
 			</h2>
 
-			<ul class="ui-tabs-nav">
-		        <li><a href="#general"><?php esc_html_e( 'General/Styling', LL_TD ); ?><span class="newred_dot">&bull;</span></a></li>
-		        <li><a href="#youtube"><?php esc_html_e( 'Youtube', LL_TD ); ?><span class="newred_dot">&bull;</span></a></li>
-		    	<li><a href="#vimeo"><?php esc_html_e( 'Vimeo', LL_TD ); ?></a></li>
-		        <?php do_action( 'lazyload_settings_page_tabs_link_after' ); ?>
-		    </ul>
+			<ul class="nav-tab-wrapper">
+				<li class="nav-tab nav-tab-active"><a href="#general"><?php esc_html_e('General/Styling', LL_TD); ?><span class="newred_dot">&bull;</span></a></li>
```

Comment:
Adding `nav-tab-active` to one tab by default is good. On the JS side though, we need to make sure it gets removed/added based on the selected tab as determined by the tab name in the URL on page load.